src/components/probeFilters.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import SelectElement from './selectElement';
+import SelectVersionElement from './selectVersionElement';
+
+
+const ProbeFilters = props => {
+  const channels = [props.channels.default, ...props.channels.valid];
+  return (
+    <div id="probe-filters">
+      <div className="form-row form-inline">
+        Filter for probes
+        <SelectElement
+          value={props.selectedProbeConstraint}
+          elementId="select_constraint"
+          items={[
+            {label: 'recorded', value: 'is_in'},
+            {label: 'new', value: 'new_in'},
+            {label: 'expired', value: 'is_expired'}
+          ]}
+          onChange={props.doProbeConstraintChange}
+        />
+        <div id="version-selection-element">
+          in version
+          <SelectVersionElement
+            value={props.selectedVersion}
+            elementId="select_version"
+            items={props.versions}
+            onChange={props.doVersionChange}
+          />
+        </div>
+        on channel
+        <SelectElement
+          value={props.selectedChannel}
+          elementId="select_channel"
+          onChange={props.doChannelChange}
+          items={channels.map(channel => {return {label: channel, value: channel};})}
```

Comment:
Alternatively, something like this (untested; syntax could be slightly off):

```javascript
items={channels.map(channel => ({label: channel, value: channel}))}
```

Comes down to style.

Reply:
But yeah, instead of using a return you can wrap the object in parens. That's valid.